internal/users/add.go

```diff
@@ -46,20 +47,18 @@ func (c *Client) Add() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			getOpts := c.Servicer.ListOptions(nil, nil)
-			createRequest := &packngo.InvitationCreateRequest{
-				Invitee:     email,
-				Roles:       roles,
-				ProjectsIDs: projectsIDs,
-			}
-			invitation, _, err := c.InvitationService.Create(organizationID, createRequest, getOpts)
+			invitationInput := metal.NewInvitationInput(email)
+			invitationInput.SetRoles(roles)
+			invitationInput.SetProjectsIds(projectsIDs)
+
+			invitation, _, err := c.InvitationService.CreateOrganizationInvitation(context.Background(), organizationID).InvitationInput(*invitationInput).Execute()
```

Comment:
It looks like this API supports `include` parameters: https://github.com/equinix-labs/metal-go/blob/main/metal/v1/api_organizations.go#L219

To match what the old code was doing with `getOpts`, this needs to be wired up to the `--include` command line flags via the servicer.